doc/source/index.rst

```diff
@@ -8,6 +8,7 @@ SUSE Enterprise Storage (SES), using Airship, shell scripts, and Ansible playboo
 .. toctree::
    :hidden:
 
+   learning/index.rst
```

Comment:
We can probably include this in contributor.rst, and increase the content on contributor.rst. Maybe we should split that contributor.rst into multiple pages.

Reply:
The new learning material has been added to the contributor section (Developer Documentation),